app/models/post.rb

```diff
@@ -2,4 +2,14 @@ class Post < ApplicationRecord
   belongs_to :author, class_name: 'User'
   has_many :comments
   has_many :likes
+
+  after_save :post_counter_update
+
+  def post_counter_update
+    author.increment(:post_counter)
+  end
```

Comment:
- [ ] Kindly make the `post_counter_update` method private so as to improve the security of your app in terms of access to this method